src/test/regress/expected/multi_follower_dml.out

```diff
@@ -21,28 +47,82 @@ SELECT * FROM the_table;
  1 | 2 | 2
 (1 row)
 
+INSERT INTO reference_table (a, b, z) VALUES (1, 2, 2);
```

Comment:
I think it might be useful to comment that `SET citus.writable_standby_coordinator TO on;`  doesn't  help to  enable writes for citus local  tables and reference tables when they are replicated to the coordinator. Because, the data is in the coordinator and will hit read-only tranaction checks on Postgres